tests/test_llmintegration.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+from dotenv import load_dotenv
+import openai
+import litellm
+import logging
+
+# Add the project root to Python path to use local agentops
+project_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, project_root)
+
+import agentops
+
+# Load environment variables
+load_dotenv()
```

Comment:
We are running tests through `tox` (`pip install tox` and run `tox` in the CLI to spin up all the tests). We're failing right now because of dotenv